source/Playnite.DesktopApp/Controls/SettingsSections/Search.xaml

```diff
@@ -40,6 +40,11 @@
                           DockPanel.Dock="Top" Margin="0,15,0,0"
                           IsChecked="{Binding Settings.IncludeCommandsInDefaultSearch}" />
 
+                <CheckBox Content="{DynamicResource SearchBoxSearchWithAcronyms}"

```

Comment:
As we discussed on Discord, this shouldn't be an option and acronym support should be always enabled.